### DISCUSSION TITLE
Not able to use ESM Modules in Next JS

### DISCUSSION THREAD

**Discussion by ShubhamTheReactGeek** [vercel/next.js#43083] [Help]
Hi

I have build one library, transpiling it to es6 modules but whenever I am using in Next JS Project. It is throwing export errors.

**Comment by GabenGar** [comment]
Do you have `"type": "module"` in your `package.json`?

**Reply by ShubhamTheReactGeek** [reply]
Yes. I had to give moduleresolution to Node16 in tsconfig to make it work.
